src/simuq/dwave/__init__.py

```diff
@@ -1 +1,2 @@
 from simuq.dwave.dwave_provider import DWaveProvider
+from simuq.dwave.dwave_sim_provider import DWaveSimProvider
```

Comment:
This class name is not clear enough to denote that it is simulated annealing; it gives the impression that it was a DWave quantum annealing simulator, which is not.